storage/shard.go

```diff
@@ -1585,6 +1593,11 @@ func (s *dbShard) Bootstrap(
 	s.bs = bootstrapping
 	s.Unlock()
 
+	var (
+		// Debug
+		numBlocksMovedToBuffer = int64(0)
```

Comment:
nit: could you make a type for this, and add methods to do the various things you're inlining for instrumentation right now. i.e.

```

type dbShardResult struct {
  numBlocksMovedToBuffer int64 
  numBlocksMerged int64 
}

func (s *dbShardResult) update(seriesBootstrapResult) {... }

func (s *dbShard) emitBootstrapResult(dbShardResult) {...}
```